src/applications/edu-benefits/0994/pages/bankInformation.js

```diff
@@ -17,7 +19,18 @@ const {
 export const uiSchema = {
   'ui:title': bankInfoTitle,
   'ui:description': bankInfoDescription,
-  bankAccount: bankAccountUI,
+  'view:bankAccount': {
+    'ui:field': ReviewCardField,
+    'ui:options': {
+      viewComponent: PaymentView,
+      reviewTitle: 'Payment information',
+      editTitle: 'Add new bank account',
+      itemName: 'account',
+      startInEdit: formData => !formData['view:hasBankInformation'],
+      volatileData: true,
+    },
+    bankAccount: bankAccountUI,
```

Comment:
Should this be a bug with us-forms-systems that the `bankAccountUI` doesn't provide masking options ?

Reply:
I'm not sure. I've always seen it masked in the app